plugin.video.vercelliwebtv/default.py

```diff
@@ -28,6 +28,8 @@ def parameters_string_to_dict(parameters):
     return paramDict
  
 def addLinkItem(parameters, li):
+    li.setProperty('IsPlayable', 'true')
+    li.setInfo('video', {})
```

Comment:
btw if you have more info like duration, audio/video codec and whatever you also set it here.